doc/source/_static/numpy.css

```diff
@@ -71,4 +72,32 @@ div.admonition-legacy>.admonition-title::after {
 
 div.admonition-legacy>.admonition-title {
   background-color: var(--pst-color-warning-bg);
-}
\ No newline at end of file
+}
+
+/* Buttons for JupyterLite-enabled interactive examples */
+
+.try_examples_button {
+  color: white;
+  background-color: var(--pst-color-info);
+  border: none;
+  padding: 5px 10px;
+  border-radius: 0.25rem;
+  margin-top: 3px;  /* better alignment under admonitions */
+  margin-bottom: 5px !important; /* fix uneven button sizes under admonitions */
+  box-shadow: 0 2px 5px rgba(108, 108, 108, 0.2);
+  font-weight: bold;
+  font-size: small;
+}
+
+.try_examples_button:hover {
+  transform: scale(1.02);
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
+  cursor: pointer;
+}
+
+.try_examples_button_container {
+  display: flex;
+  justify-content: flex-start;
+  gap: 10px;
+  margin-bottom: 20px;
+}
```

Comment:
I think the `try_examples_outer_iframe`, need a 0.4em top margin to avoid a different in alignment and jumping content:
<img width="521" alt="Screenshot 2024-06-27 at 11 27 05" src="https://github.com/numpy/numpy/assets/335567/3e3840ca-2d55-4bbe-a108-5ada2d739d8c">

Reply:
Addressed in 0547d8f87cdea3e8003d8e99bf171388bd0412e5.